source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -103,6 +103,19 @@ class OpenSampleItem {
 
         return element;
     }
+
+    cardSelected() {
```

Comment:
will rename it onCardSelected or handleCardSelected? cardSelected make me thing it's a bool return

Reply:
making this change now!